packages/algolia-mesh/mesh/getSearchSuggestionsInput.ts

```diff
@@ -18,5 +19,6 @@ export async function getSearchSuggestionsInput(
     query: search,
     hitsPerPage: 5,
     analytics: true,
+    userToken: getUserToken(context),
```

Comment:
Please check the `personalizationEnabled` config. Do we want to keep that? If so, shouldn't this be moved to another plugin (OR should we check the config value here?)

Reply:
I deleted the personalization config, while indeed this is managed in the algolia dashboard